02-JS-Intro-jQuery/activity 1 to 10/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title> JS </title>
+	<meta charset="UTF-8" />
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link HREF="css/style.css" rel="stylesheet" />
+	<script src="node_modules/jquery/dist/jquery.min.js"> </script>
+	<script type="text/javascript" src="script.js"></script>
+	<script type="application/javascript" src="https://api.ipify.org?format=jsonp&callback=getIP"></script>
```

Comment:
It was not asked to use that kind of script. Why do you need to get your ip anyway?